src/transforms/log_to_trace.rs

```diff
@@ -0,0 +1,36 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    config::{DataType, TransformConfig, TransformContext},
+    event::Event,
+    transforms::{FunctionTransform, Transform},
+};
+
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct LogToTrace;
+
+#[async_trait::async_trait]
+#[typetag::serde(name = "log_to_trace")]
```

Comment:
Do we intend for users to use this transform? I noticed it wasn't documented.

Reply:
That's a good point. I don't think this one will be very useful outside testing, the opposite, `trace_to_log` sounds more useful. After rethinking I think there is no point to add this `log_to_trace` transform, I will remove it.